src/main/io/displayport_msp_bf_compat.c

```diff
@@ -58,42 +58,44 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
     case SYM_DEGREES:
         return BF_SYM_GPS_DEGREE;
 
+/*
     case SYM_HEADING:
-        return BF_SYM_OVER_HOME;
```

Comment:
This display's a house icon, Inav's icon is a compass.

I am not sure the semantics here match.